snowbear/sql.py

```diff
@@ -40,27 +43,34 @@ def _get_dialect(con):
     elif isinstance(con, Connection):
         return con.dialect
     else:
-        raise "Cannot detect dialect from object"
+        raise ValueError("Cannot detect dialect from object")
 
 
 def read_sql_query(
     sql: str, con: Engine, index_col=None, coerce_float=True, chunksize=None
 ) -> pd.DataFrame:
-    if isinstance(con.dialect, SnowflakeDialect):
+    db_dialect = _get_dialect(con)
+    logger.info(f"starting read_sql_query(). the db dialect is '{db_dialect}'")
+    logger.debug(f"query sql: '{sql}'")
```

Comment:
we get it from both pandas and SQL in debug

Reply:
didn't follow